cmd/generator/main.go

```diff
@@ -196,12 +207,22 @@ func getReleasePlugin(ctx context.Context, client *github.Client, repositoryName
 		}
 	}
 
-	// If no plugin in existing database, attempt to download and inspect manifest.
-	if plugin == nil {
-		plugin = &model.Plugin{}
+	// If no plugin in existing database or the updated timestamp has changed, attempt to download and inspect manifest.
+	if plugin == nil || updatedAt.IsZero() || plugin.UpdatedAt.Before(updatedAt) {
+		if plugin == nil {
+			logger.Debug("no existing plugin; fetching download url")
```

Comment:
We probably don't need the text "fetching download url" given that is always printed below after this `if`